client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/InteractionSection/InteractionDetailsFields/InteractionExternalizationForm/InteractionExternalizationForm.tsx

```diff
@@ -0,0 +1,91 @@
+import React, {useEffect} from 'react';
+import {Controller, useFormContext} from 'react-hook-form';
+import {Collapse, Grid, Typography} from '@material-ui/core';
+import InteractionEventDialogFields
+    from 'models/enums/InteractionsEventDialogContext/InteractionEventDialogFields';
+import FormInput from 'commons/FormInput/FormInput';
+import Toggle from 'commons/Toggle/Toggle';
+import placeTypesCodesHierarchy from 'Utils/ContactEvent/placeTypesCodesHierarchy';
+import useFormStyles from 'styles/formStyles';
```

Comment:
Fix order of imports list
separate commons/models imports from files imports and modules imports

Reply:
Hopefully fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/commit/bd9413139437352056d0aaff99583d726152f8a1)